lob-api-public.yml

```diff
@@ -110,6 +110,12 @@ tags:
       Verify a list of US addresses.
       <div class="back-to-top" ><a href="#" onclick="toTopLink()">back to top</a></div>
 
+  - name: Campaigns
+    description: |
+      The campaigns endpoint allows you to create and view campaigns that can be used to send multiple letters or postcards.
+      The API provides endpoints for creating campaigns, updating campaigns, retrieving individual campaigns, and deleting
```

Comment:
This specifies that only individual campaigns can be retrieved, but there's an `all_campaigns` model which indicates the presence of a `list` operation. That should be added in.